content/cookbook/wallets/sign-message.md

```diff
@@ -8,22 +8,44 @@ The primary function of a keypair is to sign messages and enable verification of
 the signature. Verification of a signature allows the recipient to be sure that
 the data was signed by the owner of a specific private key.
 
-To do so, we can use the [TweetNaCl](https://www.npmjs.com/package/tweetnacl)
-crypto library:
+For Solana Web3.js v1, we can use the
```

Comment:
Let's move this v1 specific note into the tab of v1. v2 users dont care about it and dont need to see it